pages/[...slug].vue

```diff
@@ -0,0 +1,54 @@
+<!--
+name: slug
+description: this a page for dynamic content, in Nuxt we can use dynamic routes to create pages that match a pattern.
+  this page will match any pattern as it has `...` in it. This is a catch-all routes and it used for `@nuxt/content`
+-->
+
+<script setup lang="ts">
+const route = useRoute()
+
+const { data: page } = await useAsyncData(route.path, () => queryContent(route.path).findOne())
+
+const minHeight = computed(() => {
+  // TODO: fix dehydration issue
```

Comment:
This one I am not sure what that means, or how to fix it so we might also have to wait for this to merge?

Reply:
so right now if we try to get footer's height and then minus it from screen height, it gives some warnings (in dev only) because the footer instance hasn't rendered yet (in SSR) and it renders in client side (not a big deal for now but we can fix these todo's in the future)